specification/README.md

```diff
@@ -0,0 +1,17 @@
+# OpenTelemetry Semantic Conventions
+
+The Semantic Conventions define a common set of (semantic) attributes which provide meaning to data when collecting, producing and consuming it.
+The Semantic Conventions specify attribute names, types, meaning and valid values, and provide descriptions and example usage.
```

Comment:
This is a nit, but I wondered if this description of semantic conventions should be wider? I'm not sure if things like span name, span kind, metric instrument or metric unit can be subsumed under "attribute names, types, meaning and valid values".

This is how the [official OTel docs](https://opentelemetry.io/docs/concepts/semantic-conventions/) describe semantic conventions:

> OpenTelemetry defines Semantic Conventions (sometimes called Semantic Attributes) that specify common names for different kinds of operations and data. 

This is how they are described in the [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#semantic-conventions-stability), which is likely a too technical definition for this context:
> Semantic conventions define a contract between the signals that instrumentation will provide and analysis tools that consumes the instrumentation (e.g. dashboards, alerts, queries, etc.).

Reply:
I agree, but this this is intentionally a very narrow scope, as it is actually meant to cover only the backwards compatibility / stability guarantees. So this allows us to totally change the meaning & instructions for filling oute.g. http.method as long as we don't remove or rename the attribute key IIUC. See https://github.com/open-telemetry/opentelemetry-specification/pull/3225

Reply:
My intention was not to provide a precise definition of SemConv with that sentence but rather give a rough understanding for the reader. How about keeping it simple / vague here but linking the more detailed scope / definition from  https://opentelemetry.io/docs/specs/otel/versioning-and-stability/#semantic-conventions-stability here.

Something like:

> The Semantic Conventions specify among other things span names and kind, metric instruments and units as well as attribute names, types, meaning and valid values. For a detailed definition of the Semantic Conventions' scope see [Semantic Conventions Stability](https://opentelemetry.io/docs/specs/otel/versioning-and-stability/#semantic-conventions-stability). 

WDYT?